app/controllers/application_controller.rb

```diff
@@ -96,7 +98,10 @@ def assign_footer_page_links
 
   def set_flash_warning_reminder_to_update_details usr
     if usr.organisation and not usr.organisation.has_been_updated_recently?
-      msg = render_to_string(partial: "shared/call_to_action", locals: {org: usr.organisation}).html_safe
+      msg = render_to_string(
+        partial: "shared/call_to_action",
+        locals: {org: usr.organisation}
+      ).html_safe
       if flash[:warning]
```

Comment:
as we discussed in the hangout I think this could be extracted to a new method, or left - up to you here